source/ur_api.cpp

```diff
@@ -0,0 +1,3560 @@
+/*
+ *
+ * Copyright (C) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ * @file ur_api.cpp
+ * @version v0.5-r0.5
+ *
+ */
+#include "ur_api.h"
```

Comment:
I'm assuming this is here just to make sure the headers and function definitions compile? If so, I propose we delete it since we will soon be adding a null adapter implementation that also serves that purpose and is a better template for actual adapter implementations.

Reply:
Sounds reasonable, yeah. I only noticed it was missing when testing the `FetchContent` support and the CMake for this file is unconditional. The `null` adapter sounds like a more robust way of testing the header compiles 👍